package.json

```diff
@@ -13,9 +13,9 @@
     "scripts": {
         "start": "webpack-dev-server --mode development --config config/webpack.dev.js --env.NODE_ENV=development --history-api-fallback",
         "build": "webpack --mode production --config config/webpack.prod.js --env.NODE_ENV=production",
-        "typecheck": "tsc",
+        "typecheck": "tsc --incremental",
```

Comment:
What does incremental do?

Reply:
It does incremental typecheck. Please read docs sir!